AutomatedTesting/Gem/PythonTests/scripting/NodePalette_HappyPath_CanSelectNode.py

```diff
@@ -6,91 +6,92 @@
 """
 
 
-# fmt: off
-class Tests():
+class Tests:
     category_selected = ("Category can be selected", "Category cannot be selected")
-    node_selected     = ("Node can be selected",     "Node cannot be selected")
-# fmt: on
+    node_selected = ("Node can be selected",     "Node cannot be selected")
 
 
-GENERAL_WAIT = 0.5  # seconds
+NODE_CATEGORY = "Math"
+NODE_NAME = "String To Number"
```

Comment:
These can be moved to the scripting constants under more meaningful names.

EG:

```
NODE_CATEGORY_MATH = "Math"
NODE_STRING_TO_NUMBER = "String To Number"
```

Reply:
moved these over to the constants utility file